src/cargo/core/compiler/fingerprint/mod.rs

```diff
@@ -1695,6 +1720,7 @@ fn local_fingerprints_deps(
     deps: &BuildDeps,
     target_root: &Path,
     pkg_root: &Path,
+    envs: BTreeMap<String, Option<OsString>>,
```

Comment:
Why do we pass in an owned `BTreeMap` rather than a borrowed?

Reply:
done